protected/humhub/config/common.php

```diff
@@ -153,6 +153,7 @@
         ],
         'queue' => [
             'class' => \humhub\modules\queue\driver\MySQL::class,
+            'ttr' => 600,
```

Comment:
We can leave the default value here. It's just important that it could be changed.